tasks/src/automatic-speech-recognition/data.ts

```diff
@@ -51,7 +51,12 @@ const taskData: TaskDataCustom = {
 			id:          "facebook/s2t-small-mustc-en-fr-st",
 		},
 	],
-	spaces:       [],
+	spaces:       [
+		{
```

Comment:
https://huggingface.co/spaces/jeffistyping/Youtube-Whisperer was also trending. A bit more complex, but could be nice to share here. WDYT?